.gitignore

```diff
@@ -1 +1,4 @@
 target/
+project/
```

Comment:
`project/` should not be ignored, this directory exists in the project.

Reply:
Not sure why my git was showing a bunch of unrelated changes under `project/` on my local. I must have messed with something. Reverted now.